### PR TITLE
refactor/1652 - Refactor Irish Airports

### LINKED ISSUE #1652: Refactor Irish Airports
Cork (EICK) and Shannon (EINN) are currently broken due to inaccurate airspace. Dublin (EIDW) has had a lot of changes in the last AIRAC (Runway re-designation + a lot more!) and the proper airspace stratification should be added.

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves #1652 

- [x] Fix Cork Airspace
- [x] Fix Shannon Airspace
- [x] Update Dublin Procedures
- [x] Add Dublin CTA sectors

Test it [HERE](https://openscope-dev-pr-1653.herokuapp.com/)!

### CODE REVIEW

File: assets/airports/eick.json [hunk 1]
```diff
@@ -27,658 +27,746 @@
     "defaultMaps": ["Base Map", "Extended Runway Centrelines"],
     "airspace": [
         {
-            "floor": 10,
+            "floor": 50,
             "ceiling": 245,
             "airspace_class": "C",
             "poly": [
```
Comment (erikquinn) [inline assets/airports/eick.json:33]: For each airspace polygon of each airport, please use airspace's `labelPositions` property to place the labels at the appropriate position to clearly depict what altitudes are where:
```
"labelPositions": [
    [50.1, 9.25]
],
```

Currently, they're all piled up on top of each other (EICK pictured below as an example):
image

**Review by erikquinn — CHANGES_REQUESTED** [review]
Unless you were planning to get them in a different PR, the following airports' airspace do not reach the ground: 
- [ ] CYHZ
- [ ] CYOW
- [ ] EGGW
- [ ] LIPZ
- [ ] LROP

File: assets/airports/eick.json (hunk 1 above)
Comment (Fechulo) [reply assets/airports/eick.json:33]: Oh man, I didn't know that was a feature 😍

**Comment by Fechulo** [comment]
@erikquinn labelPositions added now. I'll address the remaining broken airports in a separate PR 😄

**Comment by erikquinn** [comment]
image

Multiple polygons with the same geometry and different altitudes is something we should avoid at all costs. For example, the above should be just 000-245.

**Review by erikquinn — APPROVED** [review]

**Pull request merged**